src/editor/serialize.ts

```diff
@@ -62,16 +62,20 @@ export function textSerialize(model: EditorModel) {
 }
 
 export function containsEmote(model: EditorModel) {
-    return startsWith(model, "/me ");
+    return startsWith(model, "/me ", true);
 }
 
-export function startsWith(model: EditorModel, prefix: string) {
+export function startsWith(model: EditorModel, prefix: string, caseInsensitive = false) {
```

Comment:
could we avoid double-negatives, ie. `caseSensitive = true`?

Reply:
I actually had that initially :D will toggle back